packages/matrix/helpers/index.ts

```diff
@@ -221,22 +221,19 @@ export async function logout(page: Page) {
   await expect(page.locator('[data-test-login-btn]')).toHaveCount(1);
 }
 
-export async function createRoom(page: Page, name?: string) {
-  let roomName: string;
-  await page.locator('[data-test-create-room-mode-btn]').click();
-  if (name) {
-    roomName = name;
-    await page.locator('[data-test-room-name-field]').fill(name);
-  } else {
-    roomName = await page.locator('[data-test-room-name-field]').inputValue();
-  }
+export async function createRoom(page: Page) {
   await page.locator('[data-test-create-room-btn]').click();
+  await page.locator(`[data-test-room-name]`).waitFor();
+  let roomName = await page.locator('[data-test-room-name]').textContent();
+  if (!roomName) {
+    throw new Error('room name is not found');
+  }
   await isInRoom(page, roomName);
   return roomName;
```

Comment:
Since with the changes I made, room names are always autogenerated on creation, I'm using the text from the created room to figure out the name. This works, but if there's an easier way to access this or even better, to access the room id, that would be much better @IanCal